services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/TopicCleanupService.java

```diff
@@ -248,6 +282,17 @@ private void populateDeprecatedTopicQueue(PriorityQueue<PubSubTopic> topics) {
     if (realTimeTopicDeletionNeeded.get() && !multiDataCenterStoreToVersionTopicCount.isEmpty()) {
       refreshMultiDataCenterStoreToVersionTopicCountMap(topicsWithRetention.keySet());
     }
+
+    // Check if there is dangling topics to be deleted.
+    if (apacheKafkaAdminAdapter.isPresent()
+        && System.currentTimeMillis() - danglingTopicCleanupIntervalMs > recentDanglingTopicCleanupTime) {
+      List<PubSubTopic> pubSubTopics = collectDanglingTopics(topicsWithRetention);
+      if (!pubSubTopics.isEmpty()) {
+        LOGGER.info("Find topic discrepancy: topics not in kafka but in xinfra {}", pubSubTopics);
```

Comment:
we should not expose the proprietary pubsub system name in OSS. And IMO it's more intuitive to say 
```
LOGGER.warn("Find topic discrepancy: {} is not present among all pubsub systems"
```

Reply:
Yeah, you are right. I will remove that.